### DISCUSSION TITLE
Add a "Used by" section on GitHub package repos: Any pros/cons or limitations?

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/epiverse-trace.github.io#72] [General]
Has anyone seen this box called "Used by" above the Contributors subtitle before?
![image](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/21230390/3f67d22a-253e-43b6-a1bc-907ea4e4852e)

Here is the [reference repo](https://github.com/randy3k/radian) 

Looks like an amazing addition to the repo to visualize all dependent repos that use one package as a dependencie. 

However, as far as I checked, it is not used in R package (e.g. [testhat](https://github.com/r-lib/testthat)).

Here is a [documentation page related to this](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#viewing-the-dependency-graph) for anyone that would like to look further.

I related this to the project evaluation framework and how to quantify community usage by quantifying package usage @annacarnegie @Bisaloo. 

If useful, any requirement from the dependent repo needed to make that connection visible could be also added as a community-related learning outcome as part of the contribution pathway (like in reproducible analysis training). The simplest presentation for this is github recognizing a file with [the following structure](https://github.com/Alt-Merklein/ICR-Competition/blob/main/requirements.txt).

My questions to start the discussion are:
- Are there any known pros or cons for this?
- Are there any limitations to applying this on R package-related analysis repositories?

**Comment by Bisaloo** [comment]
My understanding is that this is not included in R packages because of a technical limitation from GitHub's side. Otherwise, I agree it could be useful.

https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems

**Reply by avallecam** [reply]
That's true, thank you for pointing to the reference

I found this post on the GitHub community discussion board with a [request to support R](https://github.com/orgs/community/discussions/5291) from almost 2 years ago. Is there a way to motivate support for this?
